core/controlplane/config/templates/cluster.yaml

```diff
@@ -1243,6 +1243,11 @@ experimental:
   tlsBootstrap:
     enabled: false
 
+  # Tell Kubelet to renew its certificate as its expiration time is approaching
+  # Requires FeatureGate RotateKubeletClientCertificate=true on worker
+  RotateCerts:
```

Comment:
Could you move this to `kubelet.rotateCertificates`?

Sorry for taking your time but we're migrating away from the experimental settings because not only experimental but everything could change while we're in pre v1.0 😉

Reply:
ack, no problem